pom.xml

```diff
@@ -37,9 +37,9 @@
         <plexus.version>3.1.1</plexus.version>
         <quarkus.version>0.25.0</quarkus.version>
         <tycho.scmUrl>scm:git:https://github.com/jbosstools/jbosstools-quarkus.git</tycho.scmUrl>
-        <quarkus-jdt-site>https://download.jboss.org/jbosstools/vscode/snapshots/builds/quarkus-jdt/latest/all/repo/</quarkus-jdt-site>
+        <quarkus-jdt-site>https://download.jboss.org/jbosstools/vscode/stable/builds/quarkus-jdt/latest/all/repo/</quarkus-jdt-site>
         <lsp4mp-jdt-site>http://download.eclipse.org/lsp4mp/releases/0.2.1/repository/</lsp4mp-jdt-site>
-        <quarkus-ls.version>0.11.0-SNAPSHOT</quarkus-ls.version>
+        <quarkus-ls.version>0.10.0</quarkus-ls.version>
```

Comment:
is there a reason about the version backward?

Reply:
Yes the next version won't be out in time for the GA